README.md

```diff
@@ -1,104 +1,74 @@
+- `function/services/kta` - Supports triggering the KTA process map.
+- `function/services/s3` - Supports integration with AWS S3.
+- `function/services/secret-manager` - Supports integration with AWS Secrets Manager.
+- `function/services/sqs` - Supports integration with AWS SQS.
+- `function/services/ssm` - Supports integration with AWS SSM.
 - `template.yml` - An AWS CloudFormation template that creates an application.
-- `1-create-bucket.sh`, `2-deploy.sh`, etc. - Shell scripts that use the AWS CLI to deploy and manage the application.
-
-Variants of this sample application are available for the following languages:
-
-- Python – [blank-python](/sample-apps/blank-python).
-- Ruby – [blank-ruby](/sample-apps/blank-ruby).
-- Java – [blank-java](/sample-apps/blank-java).
-- Go – [blank-go](/sample-apps/blank-go).
-- C# – [blank-csharp](/sample-apps/blank-csharp).
-- PowerShell – [blank-powershell](/sample-apps/blank-powershell).
 
-Use the following instructions to deploy the sample application. For an in-depth look at its architecture and features, see [Blank Function Sample Application for AWS Lambda](https://docs.aws.amazon.com/lambda/latest/dg/samples-blank-nodejs.html) in the developer guide.
 
 # Requirements
-- [Node.js 10 with npm](https://nodejs.org/en/download/releases/)
+- [Node.js 14.6 or later with npm](https://nodejs.org/en/download/releases/)
 - The Bash shell. For Linux and macOS, this is included by default. In Windows 10, you can install the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) to get a Windows-integrated version of Ubuntu and Bash.
-- [The AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) v1.17 or newer.
+- [Oracledb for nodejs](https://node-oracledb.readthedocs.io/en/latest/) - Will need to follow step 2.3 onwards to set this up.
 
-If you use the AWS CLI v2, add the following to your [configuration file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) (`~/.aws/config`):
+Not mandatory but useful if using VSCode:
+- Prettier formatter extension
+- AWS Toolkit extension
 
-```
-cli_binary_format=raw-in-base64-out
-```
+# Local development setup
 
-This setting enables the AWS CLI v2 to load JSON events from a file, matching the v1 behavior.
-
-# Setup
 Download or clone this repository.
+Add an .env file containing:
 
-    $ git clone https://github.com/awsdocs/aws-lambda-developer-guide.git
-    $ cd aws-lambda-developer-guide/sample-apps/blank-nodejs
-
-To create a new bucket for deployment artifacts, run `1-create-bucket.sh`.
+   `NODE_ENV = 'local'`
 
-    blank-nodejs$ ./1-create-bucket.sh
-    make_bucket: lambda-artifacts-a5e491dbb5b22e0d
+Configure local code:
+- In `function/index.test.js` unskip the `'Should run the function handler'` test
+- In `function/db/dbPooljs` and `function/db/index.js` uncomment the lines starting with `oracledb.initOracleClient`
+- Comment lines 57 - 66 in `function/index.js`
```

Comment:
If this is a long-term strategy for local development, are we able to use our if (NODE_ENV === 'local') approach to do this for us rather than developers having to comment/remember to uncomment before commits etc